doc/themes/the-things-stack/layouts/partials/nav.html

```diff
@@ -39,3 +42,12 @@
     </div>
   </div>
 </nav>
+
+<script type="text/javascript" src="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.js"></script>
+<script type="text/javascript"> docsearch({
+apiKey: '4372282a02970d8402d43457bf95d486',
```

Comment:
Can we please not hardcode API keys?

Reply:
added to params in config.toml, but I don't know what the right way is to include it from the environment. we also want to be able to change the index for tts vs ttes